src/types.ts

```diff
@@ -255,3 +255,16 @@ export type TableData = {
   columns: Column[];
   totalsPosition: TotalsPosition;
```

Comment:
TotalsPosition from 247 line moved to above. Then should be fine.

Reply:
Yes I fixed it in the last commit